marlowe-playground-client/src/Help.purs

```diff
@@ -50,6 +54,57 @@ toHTML helpType =
 
   bodyText WalletsSimulatorHelp = text "The Wallets Simulator allows you to see how your contract will look from the point of view of users. You can create multiple wallets then transfer roles, add assets and apply transactions in individual wallets. To get started create a wallet by clicking on the '+' button at the top of the main panel."
 
+  bodyText EditorHelp =
+    div []
+      [ p_ [ text "Marlowe code can be edited and analysed in the code editor. To make things easier to read, you can format code at any time by right-clicking in the editor and selecting \"Format Code\"." ]
```

Comment:
It seems it is called "Format Document" at the moment, but "Format Code" sounds better, maybe we should change the name in the pop-up menu.

Reply:
Not sure that's possible since it's a built-in one, I'll change the PR